dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -448,69 +451,51 @@ object LagomPlugin extends AutoPlugin {
       LagomImport.component("lagom-reloadable-server") % Internal.Configs.DevRuntime
   )
 
-  private lazy val startServiceLocatorTask = Def.taskDyn {
-    if ((lagomServiceLocatorEnabled in ThisBuild).value) {
```

Comment:
This is a small change in behaviour, the `*Enabled` flags for the service locator, Kafka and Cassandra are no longer checked by their corresponding `start*` tasks, the start tasks start them regardless of whether they are enabled or not (the logic being, if you explicitly run the start task, then you must actually want it to start regardless of the configuration). Instead, the definition of the `lagomInfrastractureSettings` reads the enabled flags, and based on them, adds or doesn't add the service locator, Kafka and Cassandra to the list.